libobs/obs.h

```diff
@@ -2518,6 +2518,19 @@ EXPORT void obs_source_frame_copy(struct obs_source_frame *dst,
 /* Get source icon type */
 EXPORT enum obs_icon_type obs_source_get_icon_type(const char *id);
 
+/* ------------------------------------------------------------------------- */
+/* Functions to show notification in main UI */
+enum obs_notify_type {
+	OBS_NOTIFY_TYPE_INFO,
+	OBS_NOTIFY_TYPE_WARNING,
+	OBS_NOTIFY_TYPE_ERROR,
+};
+
+EXPORT uint32_t obs_show_notification(enum obs_notify_type type,
```

Comment:
I'm on the fence about the API returning `uint32_t` for the id. Usually, the id is passed as input, and it's in string format. This is helpful for consumers of this API, so that they are able to recreate the notification id at any point in time.

Reply:
My only concern with that is, how would you make sure every id is unique, if the caller is generating the id themselves?